docs/hub/post_install.md

```diff
@@ -27,145 +26,13 @@ lastupdated: "2021-02-20"
 ## Installation verification
 
 1. Complete the steps in [Install {{site.data.keyword.ieam}}](online_installation.md)
-2. Ensure that all pods in the {{site.data.keyword.ieam}} namespace are either **Running** or **Completed**:
-
-   ```
-   oc get pods
-   ```
-   {: codeblock}
-
-   This is an example of what should be seen with local databases installed. Some initialization restarts are expected, multiple restarts usually indicate an issue.:
```

Comment:
This is an example of what you should see when local databases are installed. Some initialization restarts are expected, but multiple restarts usually indicate a problem: